tests/components/tplink/__init__.py

```diff
@@ -214,17 +234,75 @@ def _mocked_strip_children(features=None) -> list[Device]:
     return [plug0, plug1]
 
 
+def _mocked_emeter_features(
```

Comment:
```suggestion
def _mocked_deprecated_emeter_features(
```
And adding a comment that these are just to keep the backwards compat?

Reply:
This is to test the existing energy sensors so not a deprecated method

Reply:
Renamed to `_mocked_energy_features`